.travis.d/gen.rb

```diff
@@ -178,6 +213,12 @@ def write_config(path, template, bind)
   biodome['mongo'] = CONFIG['mongo']['host']
 
   biodome['apps'].each do |app_name, app|
+    app['files']&.each do |key, file|
+      path = File.join KUBE_OUT_DIR, "#{app_name}-#{dome_name}-#{key}-configmap.yaml"
```

Comment:
make one configmap file per app / dome pair, this looks possible in: https://kubernetes.io/docs/tasks/configure-pod-container/configmap/ , the idea is that we want one configmap to be the configuration for one app and we don't care about reusing it anywhere else